src/core_plugins/kibana/public/management/sections/indices/_scripted_fields.js

```diff
@@ -99,6 +101,12 @@ uiModules.get('apps/management')
       $scope.remove = function (field) {
         $scope.indexPattern.removeScriptedField(field.name);
       };
+
+      $scope.getDeprecatedLanguagesInUse = function () {
+        const fields = $scope.indexPattern.getScriptedFields();
+        const langsInUse = _.uniq(_.map(fields, 'lang'));
+        return _.difference(langsInUse, ['expression', 'painless']);
```

Comment:
Extracting the array of supported languages into a constant somewhere would allow to reduce duplication with the field editor.